src/OperationalInsights/OperationalInsights/ChangeLog.md

```diff
@@ -19,6 +19,9 @@
 -->
 
 ## Upcoming Release
+* Added cmdlet Update-AzOperationalInsightsTable.
```

Comment:
I name the cmdlet as 'Update-AzOperationalInsightsTable' here, but its real name is 'Set-AzOperationalInsightsTable' in code implementation. What's your expected name?